.travis.yml

```diff
@@ -16,7 +16,4 @@ matrix:
       script: Rscript -e 'pkgdown::deploy_site_github()'
       skip_cleanup: true
   - r: oldrel
-  - r: 3.3
-  - r: 3.2
-  - r: 3.1
-
+  - r: 3.5
```

Comment:
If you fix the issue by which we need R >= 3.5.0, then you can list here the earlier versions we support, e.g.:

```
  - r: 3.4
  - r: 3.3
```